fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -683,7 +683,7 @@ adminSetPartitionVersion
     ;
 
 killStatement
-    : KILL (CONNECTION? | QUERY) INTEGER_VALUE
+    : KILL GLOBAL? (CONNECTION? | QUERY) (connId=INTEGER_VALUE | queryId=string)
```

Comment:
How about removing `GLOBAL` here. And `KILL QUERY queryId` means kill queryId globally, because query_id is globally unique. 
And connection_id is locally unique, the GLOBAL here is useless for it.

Reply:
Modified